packages/web/components/Technology/Details/Tabs/GeoLocation.js

```diff
@@ -0,0 +1,178 @@
+import React, { useState } from 'react';
+import { resetIdCounter } from 'react-tabs';
+import styled, { css } from 'styled-components';
+import useTechnology from '../../../../hooks/useTechnology';
+import useAuth from '../../../../hooks/useAuth';
+import * as Layout from '../../../Common/Layout';
+import GoogleMaps, { getMarkerIconByTerm } from '../../../GoogleMaps';
+import TechonologyEnums from '../../../../utils/enums/technology.enums';
+import CheckBoxField from '../../../Form/CheckBoxField';
+import { Protected } from '../../../Authorization';
+
+const Tabs = () => {
+	const context = useTechnology();
+	const { user } = useAuth();
+	const [markerFilters, setMarkerFilters] = useState([
+		TechonologyEnums.WHO_DEVELOP,
+		TechonologyEnums.WHERE_CAN_BE_APPLIED,
+		TechonologyEnums.WHERE_IS_ALREADY_IMPLEMENTED,
+	]);
+	const { technology } = context;
```

Comment:
:point_down: pq não assim? :thinking: 

```suggestion
	const { technology } = useTechnology();
	const { user } = useAuth();
	const [markerFilters, setMarkerFilters] = useState([
		TechonologyEnums.WHO_DEVELOP,
		TechonologyEnums.WHERE_CAN_BE_APPLIED,
		TechonologyEnums.WHERE_IS_ALREADY_IMPLEMENTED,
	]);
```

Reply:
Pode sim @mateus4k eu que esqueci de arrumar isso 👍